test/FunctionalTests/Microsoft.OData.Core.Tests/ODataInstanceAnnotationTests.cs

```diff
@@ -31,6 +31,16 @@ public void InstanceAnnotationNameWithoutPeriodInTheMiddleShouldThrowArgumentExc
             }
         }
 
+        [Fact]

```

Comment:
In the PR description you say:
"I wrote a test to validate that instance annotations of different kind were still written/not written accordingly."

Would you mind adding those test cases to the PR, or are they already covered in our existing test suite?